ebos/nncsorter.hpp

```diff
@@ -0,0 +1,34 @@
+// -*- mode: C++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*-
```

Comment:
this file seems to be rather superfluous. also: please put the code directly into the header for core ebos...

Reply:
No it is not. It is need to to reduce code duplication, as these functions are at two different places: ecltransmissibility.hh and eclwriter.hh.
Putting everything in the header is also not possible (without abusing the `inline` keyword) as you will get errors during linking because of multiply defined symbols.

Reply:
okay if you really insist it can stay. The compile unit must go, though...

Reply:
Sorry, but I do not understand why and how.

Reply:
opm-material, ewoms and core-ebos are currently all header only. I'll argue that this constitutes a significant improvement compared to the days when you had to recompile several modules to develop most features: e.g., you had to recompile opm-core, opm-grid and opm-autodiff after any changes you made in the transmissibility code and if you forgot some of them it resulted in *very* weird symptoms. now you can do this by only recompiling the simulator of interest. I want core-ebos to stay a "good upstream citizen".